Sources/Colors/ColorProvider.swift

```diff
@@ -416,6 +416,7 @@ public struct ColorProvider {
     /// Expandable
     var expandableTitleText: Color { token.text }
     var expandableParagraphText: Color { token.text }
+    var expandableDefaultBackground: Color { token.background }
```

Comment:
Adding a color to ColorProvider without it being part of WARP colors or tokens will remove it next time we sync with Figma/Taxonomy sheet. So you should avoid doing that.

And I beleive the default expandable background color is `.clear` not token.background which will be `.white` in light mode and `.gray900` in dark mode.

So my suggestion is remove this colors, and use .clear in your `ExpandableStyle`

Reply:
Ah, I was not aware 😄 I've changed it to `.clear` now 👌